src/common/user-store.ts

```diff
@@ -71,11 +71,12 @@ export class UserStore extends BaseStore<UserStoreModel> {
     if (kubeConfig) {
       this.newContexts.clear();
       const localContexts = loadConfig(kubeConfig).getContexts();
-      console.log(localContexts)
-      localContexts
-        .filter(ctx => ctx.cluster)
-        .filter(ctx => !this.seenContexts.has(ctx.name))
-        .forEach(ctx => this.newContexts.add(ctx.name));
+      localContexts.forEach(({ cluster, name }) => {
```

Comment:
why this change, the older version is more clear

Reply:
Not for me, sorry. Older version more like functional-style, but I would not say it's more readable (for humans) + there there is no need to run multiple cycles on what's can be updated in one run.

Reply:
I think this is matter of taste. I would revert this change as it's not related to views management at all, right?